cpp/log/cert.cc

```diff
@@ -546,12 +566,314 @@ Cert::Status Cert::ExtensionStructure(int extension_nid,
       LOG(WARNING) << "Corrupt extension data";
       LOG_OPENSSL_ERRORS(WARNING);
     }
+
     return FALSE;
   }
 
   return TRUE;
 }
 
+bool Cert::IsRedactedHost(const std::string& hostname) {
```

Comment:
This doesn't need to be a member function I think, can you make it free standing?